crates/threshold-signature-server/src/node_info/api.rs

```diff
@@ -26,3 +33,16 @@ pub async fn hashes() -> Json<Vec<HashingAlgorithm>> {
     let hashing_algos = HashingAlgorithm::iter().collect::<Vec<_>>();
     Json(hashing_algos)
 }
+
+/// message is stripped 0x keccak hex encoded hash of message
```

Comment:
This isn't really describing what the API does. I'd recommend something like this:

```suggestion
/// Returns the list of subgroup signers given a message hash.
///
/// Note: the message hash should not include a preceding `0x`.
```

I guess on a sidenote, why not allow the client to pass in the message with the `0x` and just strip it within the function? I think that would be more user friendly

Reply:
tend to like to push complexity like that off to the client, plus we are removing this code in a release anyways so rather not push it